config/main.py

```diff
@@ -7381,5 +7381,59 @@ def date(date, time):
     clicommon.run_command(['timedatectl', 'set-time', date_time])
 
 
+#
+# 'banner' group ('config banner ...')
+#
+@config.group()
+def banner():
+    """Configuring system banner messages"""
+    pass
+
+
+@banner.command()
+@click.argument('state', metavar='<enabled|disabled>', required=True, type=click.Choice(['enabled', 'disabled']))
+def state(state):
+    """Set banner feature state"""
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry(swsscommon.CFG_BANNER_MESSAGE_TABLE_NAME, 'global',
```

Comment:
>CFG_BANNER_MESSAGE_TABLE_NAME [](http://example.com/codeflow?start=35&length=29)

Why not use FEATURE table?

Reply:
FEATURE table was designed for SONiC App extensions features. The Banner feature is not the case...

Reply:
I see many features are not SONiC App extensions feature there.

Reply:
> I see many features are not SONiC App extensions feature there.

Sure. But we need to keep separate. In that case, it is much easier to handle. In hostcfgd we have a logic to handle FEATURE table changes. It would be overload to add additional logic to handle the banner there.
So for that, we added a new table and new subscriber in hostcfgd to handle it separately. The code looks much clearer and it is consistent over other features: NTP, RSYSLOG, SSH, etc.

Reply:
@qiluo-msft FEATURE table in general is associated with an application with a docker today. We have a dedicated application featured which listens to feature table and enables system services. We also have coppmgr using this logic to enable traps. Adding entry to feature table when there is no associated docker will lead to errors in some flows and currently it is not supported. Hence in this scenario it is advisable to use a separate table.

Reply:
@qiluo-msft please see @dgsudharsan answer. I agree with him